src/Serilog.Sinks.AzureDataExplorer/Sinks/AzureDataExplorerSink.cs

```diff
@@ -1,107 +1,172 @@
 ﻿using System.IO.Compression;
+using System.Runtime.CompilerServices;
 using Kusto.Data.Common;
 using Kusto.Ingest;
 using Microsoft.IO;
 using Serilog.Events;
-using Serilog.Sinks.Azuredataexplorer;
-using Serilog.Sinks.Azuredataexplorer.Extensions;
+using Serilog.Sinks.AzureDataExplorer.Extensions;
 using Serilog.Sinks.PeriodicBatching;
 
-namespace Serilog.Sinks.AzureDataExplorer
+[assembly: InternalsVisibleTo("Serilog.Sinks.AzureDataExplorer.Tests")]
+
+namespace Serilog.Sinks.AzureDataExplorer.Sinks
 {
-    internal class AzureDataExplorerSink : IBatchedLogEventSink, IDisposable
+    internal sealed class AzureDataExplorerSink : IBatchedLogEventSink, IDisposable
     {
-        private static readonly RecyclableMemoryStreamManager s_recyclableMemoryStreamManager = new RecyclableMemoryStreamManager();
-        private static readonly List<ColumnMapping> s_defaultIngestionColumnMapping = new List<ColumnMapping>
+        private static readonly RecyclableMemoryStreamManager SRecyclableMemoryStreamManager = new RecyclableMemoryStreamManager();
+
+        private static readonly List<ColumnMapping> SDefaultIngestionColumnMapping = new List<ColumnMapping>
```

Comment:
Is this duplicated in AzureDataExplorerDurableSink? can you reduce the code duplication?

Reply:
agreed, removed the duplication